### PR TITLE
Adding info to html

### PR DESCRIPTION
**Pull request by roxyrodbeck**
addition of spinach

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 </head>
 
 <body>
-    <span>[Add your name here]</span>
+    <span>Roxy</span>
```
Comment (wbryceangell) [inline index.html:11]: Maybe you could add your last name here

Comment (roxyrodbeck) [reply index.html:11]: Oh ya, good idea!

**Review by roxyrodbeck — COMMENTED** [review]
Testing one two

**Review by roxyrodbeck — COMMENTED** [review]
added spinach

**Review by wbryceangell — APPROVED** [review]

**Pull request merged**